app/lib/queues/populateIdListQueue.js

```diff
@@ -60,12 +60,15 @@ function addPageToQueue(q, pageNo) {
 }
 
 function start(totalPages, workers, drain) {
-  const q = async.queue(processQueueItem, workers);
+  if (totalPages > 0) {
```

Comment:
I know there are no unit tests for this module atm but would it be possible to add some?

Reply:
Have added some tests to improve coverage